makefile

```diff
@@ -0,0 +1,15 @@
+FRONTEND_DIR = ./web
+BACKEND_DIR = .
+
+.PHONY: all build-frontend start-backend
+
+all: start-frontend start-backend
+
+build-frontend:
```

Comment:
Seems `start-frontend` is meant to be build-frontend.

Reply:
addressed in https://github.com/Calcium-Ion/new-api/pull/79/commits/699fe256d0d85b9b7bae6dd6483fd9a2e793a579